src/systemctl/systemctl.c

```diff
@@ -4944,6 +4944,29 @@ static int print_property(const char *name, sd_bus_message *m, const char *conte
 
                         return 0;
 
+                } else if (contents[1] == SD_BUS_TYPE_STRUCT_BEGIN && streq(name, "TimersCalendar")) {
+                        const char *base, *spec;
+                        uint64_t next_elapse;
+
+                        r = sd_bus_message_enter_container(m, SD_BUS_TYPE_ARRAY, "(sst)");
+                        if (r < 0)
+                                return bus_log_parse_error(r);
+
+                        while ((r = sd_bus_message_read(m, "(sst)", &base, &spec, &next_elapse)) > 0) {
+                                char timespan[FORMAT_TIMESPAN_MAX];
+
+                                print_prop(name, "{ %s=%s ; next_elapse=%s }", base, spec,
+                                           format_timespan(timespan, sizeof(timespan), next_elapse, 0));
```

Comment:
Hmm, I get nonsensical values in output:
```console
$ systemctl list-timers unbound*
NEXT                         LEFT    LAST                         PASSED  UNIT                 ACTIVATES
Sat 2018-02-03 00:00:00 CET  8h left Fri 2018-02-02 00:00:22 CET  15h ago unbound-anchor.timer unbound-anchor.service

1 timers listed.
$  build/systemctl show -p TimersCalendar unbound-anchor.timer
TimersCalendar={ OnCalendar=*-*-* 00:00:00 ; next_elapse=48y 1month 2d 12h 30min }
$ build/busctl introspect org.freedesktop.systemd1 /org/freedesktop/systemd1/unit/unbound_2danchor_2etimer |grep TimersCalendar
.TimersCalendar                     property  a(sst)    1 "OnCalendar" "*-*-* 00:00:00" 15176…00 emits-invalidation
```

Reply:
Ouch! The value is not timespan but timestamp. I've updated.